presto-product-tests/README.md

```diff
@@ -244,6 +244,9 @@ be any one of the available profiles:
     presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,profile_specific_tests
     ```
 
+Note: SQL Server product-tests use `microsoft/mssql-server-linux` docker container. We assume you accept the license
```

Comment:
I don't know how it should sound from the legal perspective but "we assume" does not sound good to me. Maybe "By running sql server product tests you accept the license ..." is better? Or someone with actual legal knowledge should take a stand here?